test/node.js

```diff
@@ -13,6 +13,7 @@ test('Node constructor', function (t) {
       path: 'data/' + process.pid,
       to: HEIGHT
     })
+    node.on('error', console.log)
```

Comment:
We probably shouldn't do this, since it will swallow errors so that they don't cause the tests to fail.